packages/ui/atomic/template/package.json

```diff
@@ -10,13 +10,12 @@
   "dependencies": {
     "@coveo/atomic": "2.19.4",
     "@coveo/headless": "2.8.3",
-    "@coveo/search-token-lambda": "1.36.3",
-    "@stencil/core": "2.22.2"
+    "@stencil/core": "2.17.3"
```

Comment:
Mh no: if the user uses the CLI through the binaries it would install twice, and the CLI ain't lightweight enough to be 'OK' about it.

Maybe isolating the `deploy` command into its own plugin would be OK to make it lightweight enough (and avoid a cyclic dependency), but I'm not sure.